components/schema-migrator/main.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"time"
+
+	"github.com/kyma-project/control-plane/components/schema-migrator/cleaner"
+
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/golang-migrate/migrate/v4/database/postgres"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	_ "github.com/lib/pq"
+)
+
+func main() {
+	time.Sleep(20 * time.Second)
+	migrateErr := invokeMigration()
+	if migrateErr != nil {
+		fmt.Printf("while invoking migration: %s", migrateErr)
+	}
+
+	// continue with cleanup
+	err := cleaner.Halt()
+
+	time.Sleep(5 * time.Second)
+
+	if err != nil || migrateErr != nil {
+		fmt.Printf("error during migration: %s\n", migrateErr)
```

Comment:
Consider using the log package to do logging as it adds more clarity when reading the logs e.g., timestamp present